components/StyledButtons.js

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+const AddButton = styled.button`
+  background-color: #19c5fa;
+  opacity: 65%;
+  border: 1px grey solid;
+  border-radius: 50%;
+  position: fixed;
+  bottom: 90px;
+  right: 10px;
+  padding: 0.5em 0.8em;
+  font-size: 2em;
+`;
+
+const BackButton = styled.button`
+  position: fixed;
+  bottom: 100px;
+  left: 10px;
+`;
+
+const SubmitButton = styled.button`
+  padding: 0.5em;
+  width: 20%;
+`;
+
```

Comment:
Your button components do not have much logic. I think you could just go with the styled buttons and set the necessary attributes when needed. If this is not what you want, then you should put every component in a single file with the according styled component.